### PR TITLE
Employers can post a job offer.

### PR DESCRIPTION
**Pull request by joao-alegria**
Ended the implementation of this feature, as well as all the tests needed to the new code.
Also added some additional rest endpoints and the html and javascript to the visualization of the job offers.

### CODE REVIEW

File: rest_api/src/main/resources/static/assets/js/jobs.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
+//var base_api_url = "http://192.168.160.63:8080";
+var base_api_url = "http://localhost:8080";
```
Comment (aspedrosa) [inline rest_api/src/main/resources/static/assets/js/jobs.js]: This doesn't work on production. See this -> 8a6243ff939d42105798e550b21e377bd508e232

File: rest_api/src/main/resources/static/assets/js/login.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
-var base_api_url = "http://192.168.160.63:8080";
-//var base_api_url = "http://localhost:8080";
+//var base_api_url = "http://192.168.160.63:8080";
```
Comment (aspedrosa) [inline rest_api/src/main/resources/static/assets/js/login.js]: Same problem mentioned before. Here you can either make a commit fixing it or fixing it when anyone solves the existing conflicts with the master

File: rest_api/src/main/resources/static/assets/js/profile.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
+var profile = {};
+
+window.onload = function() {
+    profile["type"] = "tmp";//localStorage.getItem("type");
```
Comment (aspedrosa) [inline rest_api/src/main/resources/static/assets/js/profile.js]: Is this supposed to be like this?

File: rest_api/src/main/resources/static/assets/js/search.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
+//var base_api_url = "http://192.168.160.63:8080";
+var base_api_url = "http://localhost:8080";
```
Comment (aspedrosa) [inline rest_api/src/main/resources/static/assets/js/search.js]: Same as mentioned before.

File: rest_api/src/main/resources/static/assets/js/welcome.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
-var base_api_url = "http://192.168.160.63:8080";
-//var base_api_url = "http://localhost:8080";
+//var base_api_url = "http://192.168.160.63:8080";
+var base_api_url = "http://localhost:8080";
```
Comment (aspedrosa) [inline rest_api/src/main/resources/static/assets/js/welcome.js]: Same as mentioned before.

File: rest_api/src/main/resources/static/assets/js/profile.js (hunk 3 above)
Comment (joao-alegria) [reply rest_api/src/main/resources/static/assets/js/profile.js]: This code actually is not mine. I had to merge my branch with another one to obtain the most recent web app. But I will correct it.

File: rest_api/src/test/resources/application.properties [hunk 6]
```diff
@@ -1,5 +1,10 @@
 spring.jpa.hibernate.ddl-auto=create-drop
 spring.datasource.url=jdbc:mysql://192.168.160.63:3306/tests?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC
+<<<<<<< HEAD
 #spring.datasource.url=jdbc:mysql://localhost:3306/db
 spring.datasource.username=root
-spring.datasource.password=root
\ No newline at end of file
+spring.datasource.password=root
+=======
```
Comment (aspedrosa) [inline rest_api/src/test/resources/application.properties]: Merge not completed. Accept changes from origin

File: rest_api/.gitignore [hunk 7]
```diff
@@ -39,5 +37,12 @@ local.properties
...
 .project
 
+<<<<<<< HEAD
 # logs
+=======
+#intellij
+.idea/
+*.iml
+
+>>>>>>> origin
```
Comment (aspedrosa) [inline rest_api/.gitignore]: Merge not completed. Move `# logs` near to *.logs and keep `#intellij`

**Pull request merged**